repobuilder/job.go

```diff
@@ -327,14 +328,14 @@ func (j *repoBuilderJob) injectNewPackages(local string) (string, error) {
 }
 
 func (j *repoBuilderJob) getPackageLocation() string {
-	if j.release.IsDevelopmentBuild() {
-		// nightlies to the a "development" repo.
-		return "development"
-	} else if j.release.IsReleaseCandidate() {
-		// release candidates go into the testing repo:
+	if j.release.IsReleaseCandidate() {
+		// release candidates go into the testing repo.
 		return "testing"
+	} else if j.release.IsDevelopmentBuild() || j.release.IsContinuous() {
+		// nightlies and continuous releases to the a "development" repo.
```

Comment:
go to the "development" repo

Development releases should also go to the "development" repo.